Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md

---
id: polkadex-dashboards
title: Polkadex Dashboards
sidebar_label: Polkadex
description:

>   Polkadex is a decentralized exchange parachain on Polkadot that combines the benefits of
>   centralized and decentralized exchanges, offering a high-performance trading experience with
>   trustless custody.

Reviewer: ```suggestion
  Polkadex is a decentralized exchange that combines the benefits of
  centralized and decentralized exchanges, offering a high-performance trading experience with
  trustless custody.
```

keywords: [polkadot, dashboard, dune, polkadex, decentralized exchange]
slug: ../polkadex-dashboards
---

# Polkadex Dashboards

## Overview

Polkadex is a decentralized exchange parachain on Polkadot that combines the benefits of centralized
and decentralized exchanges, offering a high-performance trading experience with trustless custody.
It aims to provide a seamless trading environment while ensuring the security of a decentralized
platform.

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Polkadex parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Polkadex parachain:
```

- [Polkadex](https://dune.com/substrate/polkadex): This dashboard provides a comprehensive view of
  trading activities and token dynamics within the Polkadex network.

## Key Tables

Data from the Polkadex parachain is organized into several key tables:

- `polkadex.balances`
- `polkadex.blocks`
- `polkadex.calls`
- `polkadex.events`,
- `polkadex.extrinsics`
- `polkadex.transfers`

## Useful Queries

> Currently, there are no specific queries provided. Please check back later for updates.

Reviewer: ```suggestion
Currently, there are no specific queries provided.
```
